### PR TITLE
E2E fzf Test Selection Autocompletion

### PR DESCRIPTION
**Pull request by chatton**
## Description

This PR adds one more QOL improvement if `fzf` is installed. If no argument is supplied to `make e2e-test` and `fzf` is installed, you will get a dropdown list of all available tests to select from.

image

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

**Review by damiannolan — APPROVED** [review]
Awesome

**Review by DimitrisJim — APPROVED** [review]
checked this locally and it works perfectly!

**Review by crodriguezvega — APPROVED** [review]
🆒

File: e2e/README.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ make e2e-test entrypoint=TestInterchainAccountsTestSuite test=TestMsgSubmitTx_Su
 
 If `jq` is installed, you only need to specify the `test`.
 
+If `fzf` is installed, you only need to run `make e2e-test` and you will be prompted with interactive test selection.
```
[the remark this answers is not in the document]
Comment (chatton) [reply e2e/README.md:83]: good point, I can make sure the conditionals account for both since they are both required

**Pull request merged**